tools/test-build

```diff
@@ -62,6 +62,7 @@ sphinx_html()
 {
     (
         cd docs/sphinx
+        export SPHINXOPTS="-W"
```

Comment:
Why are we exporting this in the evironment when it's being passed on the command-line to ant?  Why not just use `-Dsphinx.warnopts=$SPHINXOPTS=-W` ?